iroh/src/disco.rs

```diff
@@ -432,6 +433,79 @@ const fn msg_header(t: MessageType, ver: u8) -> [u8; HEADER_LEN] {
     [t as u8, ver]
 }
 
+const TRANSACTION_ID_SIZE: usize = 12;
+
+/// The transaction ID is a 96-bit identifier, used to uniquely identify
+/// STUN transactions. It primarily serves to correlate requests with
+/// responses, though it also plays a small role in helping to prevent
+/// certain types of attacks. The server also uses the transaction ID as
+/// a key to identify each transaction uniquely across all clients.
```

Comment:
Nitpicking: rust doc conventions

```suggestion
/// STUN transaction ID.
///
/// The transaction ID is a 96-bit identifier, used to uniquely identify
/// STUN transactions. It primarily serves to correlate requests with
/// responses, though it also plays a small role in helping to prevent
/// certain types of attacks. The server also uses the transaction ID as
/// a key to identify each transaction uniquely across all clients.
```